src/components/DateInput/index.js

```diff
@@ -227,23 +225,43 @@ const DateInput = ({
 }
 
 DateInput.propTypes = {
+  /**
+   * `true` if this input need to be focused by default
```

Comment:
It is a perturbing sentence, maybe `When this value is set to true the input will be focused by default`